### DISCUSSION TITLE
how to bind another key to instead of esc in the `sudo` plugin

### DISCUSSION THREAD

**Discussion by saltfishh** [ohmyzsh/ohmyzsh#10540] [Help] [answered]
as the description of title,i wanna to press twice `L_CTRL` to trigger the `sudo` plugin,the esc key is a little far.

thanks for that:)

**Comment by mcornella** [comment, marked as answer]
The Left Control by itself does not produce a key sequence. You can try that by running `cat` and pressing a key combination. That is because Control is what is called a key modifier. You'll need to use something else instead.

Once you get the key combination you want (with the `cat` trick mentioned above), you can use the `bindkey` command to bind that key sequence to the `sudo-command-line` function:
```
bindkey -M emacs '<seq>' sudo-command-line
bindkey -M vicmd '<seq>' sudo-command-line
bindkey -M viins '<seq>' sudo-command-line
```
where `<seq>` is the key sequence you've found (without the `<>` symbols).

**Reply by saltfishh** [reply]
i see,thanks for answer!
